ticktacotoe.sublime-workspace

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
While the sublime project file is meant to be shared, the sublime workspace is best kept separate for each person (it keeps track of things like which tabs you have open). You can remove it from the repository and then add an entry to your `.gitignore` file so it doesn't get checked in when sublime re-creates it.